### PR TITLE
Fix/v3 scrollbars

### PR DESCRIPTION
**Pull request by wayfarer3130**
Just enable the CSS/HTML class to enable showing the scrollbars on the StudyBrowser thumbnail list

### CODE REVIEW

**Comment by sedghi** [comment]
I deliberately removed it as it was not in the initial design, based on the discussion if you can auto-hide it when mouse is out of the study browser panel, and it only appears if the mouse enters, I don't see why not to merge.

**Comment by wayfarer3130** [comment]
> I deliberately removed it as it was not in the initial design, based on the discussion if you can auto-hide it when mouse is out of the study browser panel, and it only appears if the mouse enters, I don't see why not to merge.

The issue is the scroll bar wasn't appearing at all, whether or not the mouse was in the area, so there wasn't a way to scroll many thumbnails.   If you have a CSS rule that shows only when hovering, I'd be fine with that, but that isn't how it works right now.

File: platform/ui/src/assets/styles/styles.css [hunk 1]
```diff
@@ -1,6 +1,11 @@
 /* CUSTOM OHIF SCROLLBAR */
 .ohif-scrollbar {
   scrollbar-color: #3E5967 transparent;
+  overflow-y: hidden;
```
Comment (wayfarer3130) [inline platform/ui/src/assets/styles/styles.css:4]: As requested, make this hidden unless hovering.
@sedghi - I did this for both the measurements and the thumbnail area, do you want me to do that just for thumbnails?

**Review by sedghi — CHANGES_REQUESTED** [review]
The UI has an undesired width change (see video below), I believe you should use [scrollbar-gutter ](https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-gutter)

https://user-images.githubusercontent.com/7490180/147146475-56c0563d-a30a-4304-8d7d-c1d842d8fcbf.mp4

)

**Comment by wayfarer3130** [comment]
@sedghi  -added the scrollbar gutter as requested.  Also, I moved the scrollbar to the thumbnail level, which keeps the header stable.

**Comment by sedghi** [comment]
@wayfarer3130 great it works. One of the tests is failing which is for snapshot modal which has a auto-hide scrollbar right now as well.

**Comment by wayfarer3130** [comment]
@sedghi  - I fixed the failing test (well, at least locally, still waiting for a CI build).  The problem is a defect in WebKit that doesn't allow scrollIntoView to execute if the overflow is hidden.  I resolved it by making the scrollbars always visible for ohif-scrollbar, and then set them to hide for the thumbnail view.

**Pull request merged**